### PR TITLE
Make it clear how to relay emails in postfix

### PR DESCRIPTION
**Pull request by worikgh**
Humbly offered.
I am beginning my journey with Mailman, and this piece of advice would have saved me some time
Keep up the good work!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -372,11 +372,9 @@ configuration: python:mailman.config.exim4
 ```
 
 
-To use [Postfix][12], it should be set up to relay emails from
-`172.19.199.2` and `172.19.199.3`. The mailman specific configuration is
-mentioned below which you should add to your `main.cf` configuration file,
+To use [Postfix][12], edit the `main.cf` configuration file,
 which is typically at `/etc/postfix/main.cf` on Debian-based operating
-systems:
+systems.  Add `172.19.199.2` and `172.19.199.3` to `mynetworks` so it will relay emails from the containers and add the following configuration lines:
```
Comment (maxking) [inline README.md]: Can you please break the line at 80 chars?

**Comment by maxking** [comment]
Thanks very much for your contribution. I added a comment for line break, but otherwise looks good to me.

Will merge once the comment is resolved.

**Comment by worikgh** [comment]
I confess to inexperience in editiing a pull request like this.  But I think it is done, hard to tell from the interface!

**Comment by maxking** [comment]
Yep, I think we are good. Will merge as soon as the CI is done :)

**Pull request merged**